fish-apple-touchbar.fish

```diff
@@ -1,4 +1,7 @@
 function fish-apple-touchbar
+    if status is-interactive
```

Comment:
```suggestion
    if not status is-interactive
```

> Since you don't need this in a non-interactive shell, it's safe to just avoid running the whole thing.

We want  to exit the `fish-apple-touchbar` function only if the shell is **not interactive**.